In-house patch to remove unnecessary dependencies from Glance's
requirements files. The specific reasons are as follows:

iso8601			Not applicable

pycrypto		Not applicable to Solaris (M2Crypto used instead)

python-keystoneclient	Not applicable

--- glance-12.0.0.0rc1/requirements.txt.~4~	2016-03-16 06:18:49.000000000 -0700
+++ glance-12.0.0.0rc1/requirements.txt	2016-03-30 02:29:36.594153408 -0700
@@ -13,7 +13,6 @@ Routes!=2.0,>=1.12.3;python_version!='2.
 WebOb>=1.2.3 # MIT
 sqlalchemy-migrate>=0.9.6 # Apache-2.0
 httplib2>=0.7.5 # MIT
-pycrypto>=2.6 # Public Domain
 oslo.config>=3.7.0 # Apache-2.0
 oslo.concurrency>=3.5.0 # Apache-2.0
 oslo.context>=0.2.0 # Apache-2.0
@@ -31,7 +30,6 @@ PrettyTable<0.8,>=0.7 # BSD
 Paste # MIT
 
 jsonschema!=2.5.0,<3.0.0,>=2.0.0 # MIT
-python-keystoneclient!=1.8.0,!=2.1.0,>=1.6.0 # Apache-2.0
 pyOpenSSL>=0.14 # Apache-2.0
 # Required by openstack.common libraries
 six>=1.9.0 # MIT
@@ -59,5 +57,4 @@ cryptography>=1.0 # BSD/Apache-2.0
 debtcollector>=1.2.0 # Apache-2.0
 
 # timeutils
-iso8601>=0.1.9 # MIT
 monotonic>=0.6 # Apache-2.0
